mitosheet/mitosheet/code_chunks/user_defined_import_code_chunk.py

```diff
@@ -4,16 +4,36 @@
 
 # Copyright (c) Saga Inc.
 # Distributed under the terms of the GPL License.
-from typing import List, Optional, Tuple
+from typing import Dict, List, Optional, Tuple
 from mitosheet.code_chunks.code_chunk import CodeChunk
-from mitosheet.types import ColumnID
+from mitosheet.types import ColumnID, UserDefinedImporterParamType
 from mitosheet.state import State
+from mitosheet.transpiler.transpile_utils import column_header_to_transpiled_code
+
+def get_transpiled_importer_params(importer_params_and_type_and_value: Dict[str, Tuple[UserDefinedImporterParamType, str]]) -> str:
```

Comment:
What if we just passed through the right values, and called the transpile on them? I think this should work -- as long as the eval is right. Add the dict test first.